.orchestra/config/components/aws-sdk.yml

```diff
@@ -0,0 +1,51 @@
+#@ load("@ytt:overlay", "overlay")
+#@ load("/lib/ninja.lib.yml", "ninja")
+#@ load("/lib/create-component.lib.yml", "single_build_component")
+
+#@yaml/text-templated-strings
+---
+#@ def aws_sdk_args():
+#@ version = "1.11.122"
+#@ source_url = "https://github.com/aws/aws-sdk-cpp/archive/refs/tags/" + version + ".tar.gz"
+license: source/LICENSE
+configure: |
+  mkdir -p "$BUILD_DIR"
+  cd "$BUILD_DIR"
+
+  extract.sh --into "$BUILD_DIR/source" --save-as "aws-sdk-(@= version @).tar.gz" "(@= source_url @)"
+
+  pushd source &> /dev/null
+    patch -p1 < "${ORCHESTRA_DOTDIR}/patches/aws-sdk-force-shared.patch"
+    "${ORCHESTRA_DOTDIR}/support/aws-sdk-download-submodules.sh"
```

Comment:
We never indented this way, let's drop this.

Reply:
Done